resources/public/api/conf/1.0/application.raml

```diff
@@ -294,15 +294,261 @@ uses:
                    LSANI-C1-residences or LSANI-C4-residences or NICTSEJO-C4-residences or LAA-C4-contact-details or HMCTS-C4-contact-details
       body:
         application/json:
-          schema: !include schemas/create-contact-details.json
-          example: !include examples/create-contact-details.json
+          schema: !include schemas/individuals/create-contact-details.json
+          example: !include examples/individuals/create-contact-details.json
       responses:
         201:
           description: The created contact details
           body:
             application/json:
-              example: !include examples/create-contact-details-response.json
-              type: !include schemas/create-contact-details-response.json
+              example: !include examples/individuals/create-contact-details-response.json
+              type: !include schemas/individuals/create-contact-details-response.json
+        400:
+          body:
+            application/json:
+              type: types.errorResponse
+              examples:
+                BadRequest:
+                  description: Invalid request
+                  value:
+                    code: INVALID_REQUEST
+
+  /organisations/self-assessment/{utr}/return/details:
+    uriParameters:
+      utr:
+        description: "A Unique Tax Reference"
+        type: string
+        required: true
+        example: "1234567890"
+    post:
+      displayName: "Create a self-assessment organisations payload"
+      is: [headers.acceptHeader]
+      securedBy: [ sec.x-application ]
+      headers:
+        Authorization:
+          description: "OAuth 2.0 Bearer Token or server token."
+          type: string
+          required: true
+          example: Bearer <BEARER_TOKEN>
+        Content-Type:
+          description: "Specifies the content-type of the body."
+          type: string
+          required: true
+          example: application/json
+      queryParameters:
+        startYear:
```

Comment:
We need to remove the start and end years as they are not applicable for SSP. They only exist in the IF schema for suture use by other consumers. 

The scenario we are interested in from the IF schema usage description is 

- Neither startYear or endYear: will return the data for the most recent 2 years (2, 1, or 0 years can be returned) before the current SA tax year (CY-1)